content/blog/posts/monitoring-dashboards-with-openobserve.md

```diff
@@ -0,0 +1,304 @@
+
+Our open-source observability platform makes monitoring simple and intuitive, with:
+
+* Easy setup
+* Powerful dashboards
+* Intuitive user interface
+
+> Follow along with our simple guide below and check out our detailed YouTube tutorial!
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/kjUvXQdL798?si=guA2AK3COvYJolIr" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+#### What to Expect?
+
+In this guide, we'll explore how to build monitoring dashboards in OpenObserve, we'll dive into the key features, best practices, and step-by-step instructions for creating customizable dashboards that meet your specific needs.
+
+### Key Features of OpenObserve Dashboards
+
+* **Real-time data visualizations**:  Automatically refresh your dashboards to monitor live streams with accuracy.
+* **Easy Dashboard Creation**: Intuitive UI and extensive panel customization.
+* **18+ Chart Types**: Visualize data with a variety of chart options, including heatmaps, tables, and time-series graphs.
+* **Customizable**: Personalize dashboards with filters, variables, and templating for tailored views.
+* **Alerting and Notification**: Set up custom alerts and notifications to stay informed of performance issues.
+* **Scalable**: Support for large-scale deployments with high-performance rendering.
+
+Now that you know the highlights, let's move on to creating your first dashboard in OpenObserve!
+
+## Getting Started: Setting Up OpenObserve
+
+### Step 1: Installing OpenObserve
+
+To get started with OpenObserve, you have several options for installation. Below are the most common methods:
+
+1. **Using Docker (Recommended for Quick Setup):**
+   If you have Docker installed on your machine, setting up OpenObserve is straightforward. Run the following command in your terminal or command prompt:
+
+ Docker images are available at [https://gallery.ecr.aws/zinclabs/openobserve](https://gallery.ecr.aws/zinclabs/openobserve)
+
+   ```bash 
+   docker run -v $PWD/data:/data -e ZO_DATA_DIR="/data" -p 5080:5080 \
+    -e ZO_ROOT_USER_EMAIL="root@example.com" -e ZO_ROOT_USER_PASSWORD="Complexpass#123" \public.ecr.aws/zinclabs/openobserve:latest
+   ```
+   *Replace `root@example.com` and `Complexpass#123` with your desired credentials.*
+
+   Once the container is up and running, navigate to http://localhost:5080 and login in your web browser to access the OpenObserve UI.
+
+2. **Self-Hosting Locally:**
+- Download the latest OpenObserve binary from the [official GitHub repository](https://github.com/openobserve/openobserve).
+- Extract the zip file to a directory (e.g., C:\OpenObserve).
+- Run `openobserve.exe`.
+
+3. **Helm (for Kubernetes):**
+- Run this commmand
+```bash
+helm repo add openobserve https://openobserve.github.io/helm-charts
+helm install openobserve openobserve/openobserve
+```
+
```

Comment:
Don't need to give all the options to install OpenOsberve. Use one and point to quickstart docs.